doc/api/http2.md

```diff
@@ -2800,13 +2800,63 @@ given newly created [`Http2Stream`] on `Http2ServerRespose`.
 The callback will be called with an error with code `ERR_HTTP2_STREAM_CLOSED`
 if the stream is closed.
 
+## Collecting HTTP/2 Performance Metrics
+
+The [Performance Observer][] API can be used to collect basic performance
```

Comment:
Should this be `[Performance Hooks][]` or should the bottom reference be changed?

Reply:
reference updated